Dockerfile.build

```
@@ -1,4 +1,4 @@
-FROM    iroha2:base
+FROM    hyperledger/iroha2-ci:dev
```

Since this docker file is also used for other tags than `dev`, it would better to remove the `dev` designation:

```suggestion
ARG TAG
FROM    hyperledger/iroha2-ci:$TAG
```

---

Excellent!